docs/modules/interaction.md

```diff
@@ -68,153 +67,177 @@ execute as @n[tag=bs.entity.interaction] run function #bs.interaction:clear_even
+    - {nbt}`string` {nbt}`compound` **executor**: Defines the entity on which the command will be executed. Can be either **"source"** for the player who performed the action or **"target"** for the interaction entity itself. A compound can also be used to target other entities:
       - {nbt}`string` **selector**: Specifies entities for the command execution (`execute as <entity>`).
       - {nbt}`bool` **lazy**: Determines whether to resolve the selector immediately (`false`) or at runtime (`true`, default). If `false`, the selector must target only one entity, replacing the selector with that entity’s UUID. Useful for optimizing execution, especially when targeting graphical representations. If multiple entities are targeted, the first one is selected.
   :::
 
 :Outputs:
   **Return**: The ID of the created event.
 
-  **State**: The listener will trigger the event once upon hover exit.
+  **State**: The interaction will trigger the event once upon hover exit.
 ```
 
 
 ```{admonition} Edge case
 :class: warning
 
-If a player logs out while hovering over the listener, the hover leave event will trigger on the executor. If the executor is the player, the event won’t be triggered again when they log back in. Be sure to account for this edge case and handle it appropriately when the player returns.
+If a player logs out while hovering over the interaction, the hover leave event will trigger on the executor. If the executor is the player, the event won’t be triggered again when they log back in. Be sure to account for this edge case and handle it appropriately when the player returns.
 ```
 
 ::::
 :::::
 
-*Glow the listener's icon while the listener is hovered and unglow it when it is not hovered anymore:*
+*Glow the interaction's icon while the interaction is hovered and unglow it when it is not hovered anymore:*
 
 ```mcfunction
 summon minecraft:interaction ~ ~ ~ { Tags: ["bs.entity.interaction"], width: 1f, height: 1f }
 summon block_display ~-.5 ~ ~-.5 { Tags: ["bs.entity.block_display"], width: 1f, height: 1f, block_state: { Name: "minecraft:slime_block" }}
 
-# Callback to glow the icon when the listener is hovered
+# Callback to glow the icon when the interaction is hovered
 execute as @n[tag=bs.entity.interaction] run function #bs.interaction:on_hover_enter { run: "function #bs.interaction:callback/glow", executor: { selector: "@e[tag=bs.entity.block_display]", lazy: false } }
-# Callback to unglow the icon when the listener is not hovered anymore
+# Callback to unglow the icon when the interaction is not hovered anymore
 execute as @n[tag=bs.entity.interaction] run function #bs.interaction:on_hover_leave { run: "function #bs.interaction:callback/unglow", executor: { selector: "@e[tag=bs.entity.block_display]", lazy: false } }
 ```
 
 > **Credits**: Aksiome, theogiraudet
 
 ---
 
+### Default callbacks
```

Comment:
Maybe "Predefined callbacks" which is a term used in bs.block (for registries) and bs.move (collision callbacks). Also this way it appears last even in alphabetical order